docs/graphcache/errors.md

```diff
@@ -7,7 +7,7 @@ order: 7
 
 **This document lists out all errors and warnings in `@urql/exchange-graphcache`.**
 
-Any unexpected behaviour, condition, or error will be marked by an error or warning
+Any unexpected behaviour, condition or error will be marked by an error or warning
 in development, which will output a helpful little message. Sometimes however, this
 message may not actually tell you everything about what's going on.
```

Comment:
```suggestion
Any unexpected behaviour, condition, or error will be marked by an error or warning
in development. This will output as a helpful little message; sometimes, however, this
message may not actually tell you about everything that's going on.
```

Reply:
Removing the semicolon since the "however" is already a conjunction and hence not omitted, which is what would allow us to use a semicolon.